rules/from-map.js

```diff
@@ -76,18 +76,18 @@ module.exports = {
                             }
                             // The original map callback from Array.from gets nested as a parameter to the callback from map.
                             const lastCallback = getCallback(mapCallback, mapThisArgument, `${firstCallback}${restParameterString}`),
-                                restParameters = sourceCode.getText().slice(callback.end, parent.end);
+                                restParameters = sourceCode.getText().slice(callback.loc.end.column, parent.loc.end.column);
```

Comment:
This should use `callback.range[1]` instead of `loc.end.column`. Same goes for all ranges. `range[0]` is the start, `range[1]` is the end. I believe the current approach only works because all unit tests only operate on the first line.